### DISCUSSION TITLE
ChatGPT, Institutional Restrictions & Copyrights

### DISCUSSION THREAD

**Discussion by paul-fabian** [lifan0127/ai-research-assistant#145] [General]
Hi @all, maybe this isn´t the best place to ask, but at the moment we are trying to get to a common understanding on how and when to use AI in our institution. Through our institution we have access to a large pool of scientific literature, but those are not open source.
I´m wondering if this is an issue with zotero/chatGPT since I have most papers in my library as a local pdf. What information is ChatGPT sending out?
We´re also starting to try and use GPT4all for instance, which enables us to process non-anonymised data from individuals (like names, birthdates, mailadresses). Is it possible to connect Zotero with GPT4all? As far as I know, this also uses GPT4o, but with the same issue that it sends info to openai, which is prohibited by our institution. Is there any tipp you could give me on this?

Thanks in advance!

**Comment by lifan0127** [comment]
Hi @PFabian84 Although non-OpenAI models are not officially supported, some of them might work if they use OpenAI compatible APIs. If you use such a model, you can update the API base URL to point to your model server.

![image](https://github.com/user-attachments/assets/cc708b87-8fbc-4a68-869b-8fe39e5bfaf3)
